tools/roslaunch/test/xml/test-arg-all.xml

```diff
@@ -0,0 +1,48 @@
+<launch>
+
+  <arg name="required" />
+  <arg name="optional" default="not_set" />
```

Comment:
this value `"not_set"` is the same as the [upstream test-arg-include.xml](https://github.com/ros/ros_comm/blob/add_pass_all_args/tools/roslaunch/test/xml/test-arg-include.xml#L7). Should we use a different value here?